### PR TITLE
solution for the issue:"getting error while running npm run start"

### LINKED ISSUE #2: fetch data from api

### PR DESCRIPTION
**Pull request by dexta11**
Fixes #2

Changes: updated installation document for the installation on  local machines.

### CODE REVIEW

File: docs/INSTALLATION_LOCAL.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # How to install the Open Event Web App Generator on my local machine
```
Comment (aayusharora) [inline docs/INSTALLATION_LOCAL.md]: @dexta11 I don't know what does `my machine` refers to user. It should be OS specific.
`How to install the Open Event Web App Generator on my local machine`

File: docs/INSTALLATION_LOCAL.md [hunk 2]
```diff
@@ -20,10 +24,19 @@ Run the app
...
+
 or
 ```shell
 npm run server.generator
-```
+````
+
+if getting an binding error do the following:
+```shell
+npm -rf rm node_modules
```
Comment (championswimmer) [inline docs/INSTALLATION_LOCAL.md]: Is this line correct ?

File: docs/INSTALLATION_LOCAL.md (hunk 1 above)
Comment (dexta11) [inline docs/INSTALLATION_LOCAL.md]: @ayusharora I will take care for thT

File: docs/INSTALLATION_LOCAL.md (hunk 2 above)
Comment (dexta11) [inline docs/INSTALLATION_LOCAL.md]: i have committed the changes and created a new pull request kindly check.

**Pull request closed without merging**